pinot-broker/src/main/java/org/apache/pinot/broker/routing/instanceselector/InstanceSelector.java

```diff
@@ -56,7 +55,7 @@
    * to selected server instance hosting the segment and a set of unavailable segments (no enabled instance or all
    * enabled instances are in ERROR state).
    */
-  SelectionResult select(BrokerRequest brokerRequest, List<String> segments);
+  SelectionResult select(List<String> segments);
```

Comment:
Don't change this API. The future implementation of this might need the query to correctly select the instance